Automattic/woocommerce-payments
# refactor: max width on settings layout

Fixes #

#### Changes proposed in this Pull Request









Setting a max width on the settings pages.
Adding a `SettingsLayout` component to use on both Express Payments and Main Settings pages (which also includes the banner).

For background, see Jarek's comments here: p1622132280144800-slack-CU6SYV31A

#### Testing instructions





- Ensure you have the "grouped settings" flag enabled in WCPay Dev Tools
- Go to your WCPay Home: http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout
- You got elements encased in a max-width.


![2021-05-27 16 32 44](https://user-images.githubusercontent.com/273592/119900648-69c49080-bf0a-11eb-8cb5-dbf1649c27d5.gif)



-------------------
